### PR TITLE
[ADD] sale_credit_point

### PR DESCRIPTION
**Pull request by simahawk**
Excerpt from README:

> Sale based on partners' credit point.
> 
> Use case
> -------------
> 
> You have employees that have some credit (up to you where this come from)
> and you allow employees to buy products based on this credit.
> 
> Products are sold by points rather than money.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Why not using `credit_limit` + `partner_sale_risk` (or an extension over it)?

File: sale_credit_point/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 Camptocamp SA
```
Comment (mpanarin) [inline sale_credit_point/models/sale_order.py]: only 2017? ;)

**Review by mpanarin — APPROVED** [review]

**Comment by simahawk** [comment]
@pedrobaeza are you talking about these?

1. https://github.com/OCA/partner-contact/tree/9.0/partner_financial_risk
2. https://www.odoo.com/apps/modules/11.0/partner_credit_limit/

If yes:
a) I didn't knew about them but anyway sound a bit over-complicated for the goal here
b) credit limit is not OCA as far as I see
c) this is a module that we already had and that I cleaned up for sharing.

If ppl won't like this solution I can move it to our repo ;)

**Comment by pedrobaeza** [comment]
I'm talking about 1. that is also migrated to 10.0 in https://github.com/OCA/partner-contact/pull/442 and extended for sale in https://github.com/OCA/partner-contact/tree/9.0/partner_sale_risk.

They are more complete/complicated than yours, but the use case fits exactly for them (changing the `credit_limit` field for `credit_point`), so I think they duplicate the functionality. Why don't you try the other and tell me back?

**Comment by simahawk** [comment]
@pedrobaeza I'll do (next week probably) thanks. Can you set WIP / needs review / whatever label you think fits better here? tnx :)

**Comment by pedrobaeza** [comment]
OK, I set "Work in progress", as we don't have a label "Pedro is again complaining about something" :stuck_out_tongue_winking_eye:

**Comment by rafaelbn** [comment]
Hi @simahawk ! are you going to fix this PR? Soon @OCA-git-bot will close it :smile:

**Comment by rousseldenis** [comment]
@simahawk Is this still applicable ?

**Comment by simahawk** [comment]
@rousseldenis well, I don't know. It's on an old project... I'll check. I'll rebase meanwhile to see what happens w/ tests.

**Comment by rousseldenis** [comment]
@simahawk So, closing as it won't be fixed

**Pull request closed without merging**